### ISSUE TITLE
ktlint not able to add spaces to indentation

### ISSUE THREAD

**Issue by henrik242** [shyiko/ktlint#116]
I'm using gradle-kotlinter-plugin 1.6.0 that uses ktlint 0.13.0, and I guess indentation should be fairly trivial to fix?

    > Task :shared-http-client:formatKotlinMain 
    Format could not fix > src/main/java/Http.kt:56:1: Unexpected indentation (4) (it should be 8)

Ref. https://github.com/jeremymailen/kotlinter-gradle/issues/30

**Comment by MyDogTom** [comment]
@henrik242 could you share that part of code?

**Comment by henrik242** [comment]
Here's an extract from one of our classes.  ktlint was unable to correct indentation before the entries in the second constructor (lines 10-12, 13-15).

```
package no.finn.travel.fhh.api.feedback

import com.google.gson.GsonBuilder
import com.google.gson.JsonDeserializationContext

data class Notification(@JvmField val id: Long,
                        @JvmField val rateGuest: Boolean,
                        @JvmField val token: String) {
    constructor(
        id: Long,
        recipientSpidId: String,
        hostSpidId: String,
        token: String
    ) : this(
        id,
        recipientSpidId != guestSpidId,
        token
    )
}
```

**Comment by MyDogTom** [comment]
@henrik242 Thanks. As workarround you can use this formatting:

```kotlin
    constructor(id: Long,
                         recipientId: String,
                         hostId: String,
                         token: String
    ) : this(

```

@shyiko I'll prepare PR with fix.

**Comment by henrik242** [comment]
Formatting manually is not a problem,  I just wanted the automatic formatter to do its work :)

**Comment by MyDogTom** [comment]
Unfortunately, this is not completely implemented yet. I've just found out that rule was completely disabled on my codebase, that's why I was fooled during testing :( I'll investigate that further today and/or on weekend.

**Comment by MyDogTom** [comment]
@henrik242 You can try `IndentationRule` by using snapshot KtLint version - https://github.com/shyiko/ktlint#development
Alternatively, you can include https://github.com/MyDogTom/ktlint-custom-rules . This is just full copy of the `IndentationRule` from `master`, which is not released yet. 
I would really appreciate your feedback about how rule works on your code base.

**Comment by henrik242** [comment]
@MyDogTom I tried the snapshot KtLint version, and it seems to be able to fix things automatically :)
